rebar.config

```diff
@@ -1,14 +1,12 @@
- % -*- mode: Erlang;-*-
-{erl_opts, [debug_info, {i,"./_build/default/plugins/gpb/include/"}]}.
+%% -*- mode: erlang -*-
+{erl_opts, [debug_info]}.
 {plugins, [{rebar3_gpb_plugin, "1.7.2"}]}.
-{provider_hooks, [
-                  {pre, [{compile, {protobuf, compile}}]}
-                 ]}.
+{provider_hooks, [{pre,  [{compile, {protobuf, compile}}]},
+                  {post, [{clean,   {protobuf, clean}}]}]}.
```

Comment:
this will remove generated .erl & .hrl for model.proto whenever rebar3 clean executed, right?